### ISSUE TITLE
Will this project support Windows ?

### ISSUE THREAD

**Issue by meepeek** [sigmeus/meteor-desktop-app#7]

**Comment by Sivli-Embir** [comment]
:+1: That would be a great way to bypass the "no browser updates / we us IE 8 only" corporate/school restrictions.

**Comment by fix** [comment]
I agree, but the original package is not yet supporting windows. As soon as done, it will be updated
